provisioning/roles/ruby/tasks/main.yml

```diff
@@ -24,8 +27,28 @@
     dest: ~/.bashrc
     line: eval "$(rbenv init -)"
 
-- name: Install ruby {{ruby_version}}
-  shell: ~/.rbenv/bin/rbenv install {{ruby_version}}
+- name: Install ruby {{ ruby_version }}
+  command: rbenv install {{ ruby_version }}
+  args:
+    chdir: ~/.rbenv/bin/
```

Comment:
Instead of `chdir`, specify the full path to the executable: `command: ~/.rbenv/bin/rbenv install {{ ruby_version }}`.